sql/sql.y

```diff
@@ -187,9 +187,8 @@ predict_clause
 ;
 
 column_clause
-: COLUMN columns 				{ $$ = map[string]exprlist{"" : $2} }
-| COLUMN columns FOR IDENT 			{ $$ = map[string]exprlist{"" : $2} }
-| column_clause COLUMN columns 			{ $$[""] = $3 }
+: COLUMN columns 				{ $$ = map[string]exprlist{"feature_columns" : $2} }
+| COLUMN columns FOR IDENT 			{ $$ = map[string]exprlist{$4 : $2} }
 | column_clause COLUMN columns FOR IDENT 	{ $$[$5] = $3 }
```

Comment:
This rule accepts the case that the first COLUMN doesn't have FOR but the subsequent COLUMNs have FORs, like the following example:

```
COLUMN a, b COLUMN b, c FOR in1 COLUMN d, f FOR in2
```

However, I think it is alright at this time. If we realize we do need more strict grammar rules, we can change later.